src/content/api/cli.md

```diff
@@ -82,6 +82,8 @@ Name of template to generate.
 
 To generate a project without questions. When enabled, default answer for each question will be used.
 
+Know more about the questions aksed by `init` [here](https://github.com/webpack/webpack-cli/blob/master/INIT.md#description-of-questions-asked-by-the-generator)
```

Comment:
```suggestion
T> Click [here](https://github.com/webpack/webpack-cli/blob/master/INIT.md) to see the full documentation of webpack init command.
```
I would prefer to link to the whole file instead of a link fragment.